lib/pages/pool/AddLiquidity/AddLiquidity.dart

```diff
@@ -237,92 +225,220 @@ class _AddLiquidityState extends State<AddLiquidity> {
+                            width: 40,
+                            decoration: boxDecoration(Colors.transparent, 100,
+                                0.5, Colors.grey[400]!),
+                            child: TextButton(
+                                onPressed: () {},
+                                child: Text("MAX",
+                                    style: textStyle(
+                                        Colors.grey[400]!, 8, false)))),
+                        //Amount input box
+                        ConstrainedBox(
+                          constraints:
+                              BoxConstraints(maxWidth: tokenContainerWdt * 0.5),
+                          child: IntrinsicWidth(
+                            child: TextFormField(
+                              controller: tokenAmountController,
+                              onChanged: (token0Input) {
+                                bloc.add(Token0InputChanged(
+                                    double.parse(token0Input)));
+                              },
+                              style: textStyle(Colors.grey[400]!, 22, false),
+                              decoration: InputDecoration(
+                                hintText: '0.00',
+                                hintStyle:
+                                    textStyle(Colors.grey[400]!, 22, false),
+                                contentPadding: const EdgeInsets.all(9),
+                                border: InputBorder.none,
+                              ),
+                              inputFormatters: [
+                                FilteringTextInputFormatter.allow(
+                                    (RegExp(r'^(\d+)?\.?\d{0,6}'))),
+                              ],
                             ),
                           ),
                         ),
-                        Container(width: 10),
-                        Expanded(
-                          child: Text(tkr,
-                              style: textStyle(Colors.white, 16, true)),
-                        ),
-                        Icon(Icons.keyboard_arrow_down,
-                            color: Colors.white, size: 25)
                       ],
-                    ),
+                    ))
+                  ],
+                ),
+                showBalance(tknNum),
+              ],
+            ),
+          );
+        }
+
+        Widget addLiquidityToolTip(double elementWdt) {
+          return Tooltip(
+            height: 50,
+            padding: EdgeInsets.all(10),
+            verticalOffset: -100,
+            // preferBelow: false,
+            decoration: BoxDecoration(
+                color: Colors.grey[800],
+                borderRadius: BorderRadius.circular(25)),
+            richMessage: TextSpan(
+                text:
+                    "*Add liquidity to earn 0.25% of all trades on this pair proportional to your share of the pool and receive LP tokens.",
```

Comment:
Should we have the asterisk here at the beginning? I looked at other tooltip examples and they do not contain a special character like that.